.circleci/config.yml

```diff
@@ -324,9 +324,9 @@ workflows:
       - ui-instrumentation-tests:
           requires:
             - prepare-and-assemble
-      - pitest-validate:
-          requires:
-            - unit-tests
+#      - pitest-validate:
```

Comment:
Pitest is broken and has been turned off. It's not critical as we don't use this tool. I'll cut a follow-up issue for that.